packages/forma-36-react-components/src/components/Card/AssetCard/AssetCard.css

```diff
@@ -24,7 +24,7 @@
 }
 
 .AssetCard--size-small .AssetCard__asset {
-  height: calc(1rem * (142 / var(--font-base-default)));
+  height: calc(1rem * (151 / var(--font-base-default)));
```

Comment:
No chance we can get rid of these magic numbers? Isn't this just a weird way to hardcode 151px?

Reply:
I don't like it as well, but I didn't want to change the way it works now. I'd be up for a revamp in v4